packages/dbos-cloud/cli.ts

```diff
@@ -308,6 +309,26 @@ applicationCommands
     process.exit(exitCode);
   });
 
+applicationCommands
+  .command('resource-usage')
+  .description("Query the resource usage for your applications. If no time range is provided, queries for a recent completed 1-minute interval of data.")
+  .option(
+    '-s, --since <string>',
+    'UTC time since which to start querying (formatted as 2006-01-02 15:04:05.000000). Defaults to the start of a 1-minute interval ~2 minutes ago',
```

Comment:
Why the `since` and `upto` defaults are set to the same? Should we set `since` to 5 min ago and `upto` to the current time?